src/data/text/move_descriptions.h

```diff
@@ -2910,6 +2910,38 @@ static const u8 sSURGING_STRIKESDescription[] = _(
     "Mastering the Water style,\n"
     "strikes with 3 critical hits.");
 
+static const u8 sTHUNDER_CAGEDescription[] = _(
+    "Traps the foe in a cage of\n"
+    "electricity for 2 to 5 turns.");
```

Comment:
I wouldn't be against that, but wouldn't it be better to do it on a separate PR and also take care of its clones as well? :eyes:

Maybe even check and take other move or ability descriptions into account on situations where a preprocessor directive changes their behavior to a past generation's standard too.

Reply:
This just dawned on me now, but it would also be worth discussing if that treatment would apply to Thunder Cage too in any case, because as much of a clone of Fire Spin as it may be, it was still a move introduced the the latest 2 Pokémon games and behaves in the way Game Freak wanted it to behave at the time of its implementation.

What I mean is that maybe Thunder Cage should not be affected by an `ifdef` that makes Fire Spin works to some other past standard. Or rather, that it's worth discussing whether it should be affected or not.

Reply:
Yeah, I think it's worth discussing at some point.